### DISCUSSION TITLE
[Kanban] Horizontal column scrolling and no column wrapping

### DISCUSSION THREAD

**Discussion by AshRubigo** [BurningTreeC/tiddlywiki-muuri#10] [Q&A] [answered]
With respect to Muuri Kanban, is it possible using CSS to scroll through the columns with the horizontal scrollbar instead of them wrapping?

I've tried a lot of different things, but there is something about the plugin that causes what would normally work to either be ignored or to add unwanted gaps between the columns. The columns really reeeeally like to wrap. For example, I would have thought `flex-wrap: nowrap;` would do it, but that is not the case.

**Comment by BurningTreeC** [comment, marked as answer]
Hi @AshRubigo,

you would need to create the tiddler `$:/config/kanban/horizontal` and set its text to `yes`
then you would need to give the board columns a fixed size by editing `$:/plugins/BTC/Muuri-Kanban/css` (on the very top of it there's the style definition for `.tc-board-column`)
and the `.tc-board-container` would need `overflow-x: scroll`

**Reply by AshRubigo** [reply]
Beautiful, worked perfectly straight away, thank you!

**Reply by BurningTreeC** [reply]
Glad I could help you!
